### ISSUE TITLE
Insufficient screen reader support

### ISSUE THREAD

**Issue by bragefuglseth** [bragefuglseth/fretboard#11]
While the visual nature of Fretboard brings some limitations to how conveniently it can be used with screen readers, I believe that the current screen reader support isn't good enough nonetheless. The guitar neck spin button should at least have its value read aloud upon changing it. This would make the app accessible to those who can visually parse the chord pattern, but are unable to read the fret position.

**Comment by bragefuglseth** [comment]
This has been done now.
